### DISCUSSION TITLE
Dúvida sobre inicializar o XAMPP para MySQL

### DISCUSSION THREAD

**Discussion by Daniel-Ralvs** [Daniel-Ralvs/Grafhy#2] [Q&A] [answered]
Preciso rodar um código PHP mas para isso preciso ativar o XAMPP, como faço isso?

**Comment by Daniel-Alvarenga** [comment, marked as answer]
Você precisa abrir o Painel do XAMPP (pode encontrá-lo dando windows +s  e buscando por XAMPP) e clicar no botão start de MySQL

**Comment by Daniel-Ralvs** [comment]
E agora eu preciso criar uma conexão usando MySQLi no phpp, é possível?

**Comment by Daniel-Alvarenga** [comment]
Sim, você pode criar uma conexão usando as seguintes linhas adaptadas aos nomes de sua host, usuário, senha e banco de dados:

<?php

$servername = "localhost";
$username = "seu_usuario";
$password = "sua_senha";
$database = "seu_banco_de_dados";

// Criar uma conexão
$conn = new mysqli($servername, $username, $password, $database);

// Verificar a conexão
if ($conn->connect_error) {
    die("Conexão falhou: " . $conn->connect_error);
}
echo "Conexão bem-sucedida";

?>
